### PR TITLE
Start zram-touch on android-container

### LINKED ISSUE #965: Increace swap space
Hi,

After upgrading from OTA-3 to OTA-5, Unity has started crashing often. uNav, Reddit webapp, browser invokes crash for whole destkop - Untiy reboots, showing loading dots...

I've noticed that swap space is only 32MB. I believe in Canonical times it was around 500MB, but I might recall badly.

Anyway, I did experiment:

-   Rebooted so that OS memory would be as clear as possible.
-   ssh'ed into it and watched `free -m` output - it started with about (only!) 270MB available memory.
-   Launched uNav, and kept scrolling _as if_ I was driving. Available memory started to shrink.
-   After some time, available memory got down to tenths of megabytes, and whole Untiy crashed!

Then, I've increaced swap size:
```
cd /userdata/
sudo swapoff SWAP.img
sudo fallocate -l 1000000000 SWAP.img
sudo mkswap SWAP.img
sudo swapon SWAP.img
```

After this, uNav did not reproduce system crash. I see about ~180MB of swap used up constantly. System looks more stable, at least for now. 32MB default swap is really unrealistic on this device.

Please increase default swap size at least on small-memory devices such as BQ E5.

### PR DESCRIPTION
**Pull request by UniversalSuperBox**
For whatever reason, 'started lxc-android-config' isn't getting emitted on xenial. 'android-container' is emitted, though, so we can start this job at the appropriate time.

This fixes ubports/ubuntu-touch#965

### CODE REVIEW

**Comment by UniversalSuperBox** [comment]
This is approved, but we'll wait until after OTA-6 drops to merge it.

**Review by mariogrip — APPROVED** [review]
:+1:

**Pull request merged**
